pkg/util/dynamichelper/generate.go

```diff
@@ -4,5 +4,5 @@ package dynamichelper
 // Licensed under the Apache License 2.0.
 
 //go:generate rm -rf ../mocks/$GOPACKAGE
-//go:generate go run ../../../vendor/github.com/golang/mock/mockgen -destination=../mocks/$GOPACKAGE/$GOPACKAGE.go github.com/Azure/ARO-RP/pkg/util/$GOPACKAGE Interface
+//go:generate go run ../../../vendor/github.com/golang/mock/mockgen -destination=../mocks/$GOPACKAGE/$GOPACKAGE.go github.com/Azure/ARO-RP/pkg/util/$GOPACKAGE Interface,GVRResolver
```

Comment:
I see that we generate a mock for `GVRResolver`, but do not use it in this PR. We should either use `mockGVRResolver` or a mock generated by GoMock.